app/controllers/admin/users_controller.rb

```diff
@@ -35,5 +35,12 @@ def update
     end
 
     def edit; end
+
+    private
+
+    def user_params
+      params.require(:user).permit(:email, :name, :email_public, :biography, :nickname, :affiliation, :is_admin, :username, :login, :is_disabled,
```

Comment:
Updating users is only allowed by admins, right? Because if this would be possible for other roles we have to think about the is_admin attribute!